lib/docker.js

```diff
@@ -51,7 +51,8 @@ function runImage(imageId, cwd, command, mount) {
   var mountDir = crossPlatformCwd(cwd);
   var mountComponent = mount ? `-v "${mountDir}:/app/src"` : '';
   var envArgComponent = directory.getFormattedEnvArgComponent(cwd);
-  var runCommand = `run -w /app/src -p 3000:3000 --rm -it ${mountComponent} ${envArgComponent} ${imageId} sh -c "${command}" || true`;
+  var commandWithPort = command.replace("$PORT", 3000).replace("${PORT}", 3000)
+  var runCommand = `run -w /app/src -p 3000:3000 --rm -it ${mountComponent} ${envArgComponent} ${imageId} sh -c "${commandWithPort}" || true`;
```

Comment:
If escaping $PORT works, instead of replacing `$PORT` with `3000` here, can we just replace `$` with `\$` and generalize the same effect? So `$WEB_CONCURRENCY` and `$FOO` would both work too?

```js
var escapedCommand = command.replace('$', '\$');
```